guacamole/src/main/webapp/app/navigation/services/userPageService.js

```diff
@@ -60,9 +60,13 @@ angular.module('navigation').factory('userPageService', ['$injector',
      * @returns {PageDefinition}
      *     The user's home page.
      */
-    var generateHomePage = function generateHomePage(rootGroups) {
+    var generateHomePage = function generateHomePage(rootGroups,permissions) {
```

Comment:
Please include a space following the comma before the `permissions` parameter.

Reply:
The newly-added `permissions` parameter needs to be documented.

Reply:
Styled and documented.